frontend/src/pages/Frontpage/FrontpageContainer.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+import getPosts from '../../services/frontpageService';
+
+const FrontpageContainer = ({ children }) => {
+  const [retrievedPosts, setRetrievedPosts] = useState([]);
+  const [postsToDisplay, setPostsToDisplay] = useState([]);
+
+  useEffect(() => {
+    async function getPostsOnLoad() {
+      const response = await getPosts();
+      if (response.response === 'ok') {
```

Comment:
This isn't part of the API spec anymore. We can just assume that if `getPosts` didn't raise an error that the response was OK.